OpenKeychain/src/main/res/layout/redirect_import_keys_activity.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
this entire layout isn't really used, is it? can we just use a transparent activity to show the dialog instead?